src/drt/src/dr/FlexGridGraph.h

```diff
@@ -1009,6 +1011,7 @@ class FlexGridGraph
   frVector<frLayerNum> zCoords_;
   frVector<frCoord> zHeights_;  // accumulated Z diff
   std::vector<dbTechLayerDir> layerRouteDirections_;
```

Comment:
I suspect the only reason that dbTechLayerDir is slower is because the ctor is out of line and so the == comparison requires an out of line call.   I suspect inlining dbTechLayerType(Value value) would solve the problem just as well.

Reply:
Good idea. It turns out the copy constructor is the one we needed to inline. I ended up inlining all the simple constructors in both dbTechLayerDir and dbTechLayerType.